### ISSUE TITLE
How to save image results of the generator

### ISSUE THREAD

**Issue by bakermeoe** [clementabary/travelgan#1]
Thank you for making write this pytorch code. I found difficulty to plot or save the image in the jpg file. I have checked the Generator class and it doesn't have the function that can show or output the result. How can I do that? Thank you very much

**Comment by Kitty-sunray** [comment]
Did you find the solution please? How to test images?

**Comment by bakermeoe** [comment]
Hi @Kitty-sunray , last night I tried to solve your issue from my repo, i don't have time in the weekday, but i will try to answer you again in the night. As I remember, for this issue, I have solved it and implement it in my project. But to make sure, let me reply you with better answer for your issue later. 

My suggestion for you now, please use the same dataset that the project provided you (CelebA and Emoji/Bitmoji) to try, because the size of image will affect the calculation of loss sometime

And for your second question, how to test images: Testing and training use the same method, the difference only if you test an image, it means you use your training model result. I think I have provided the model of 500 epoch result in the project. So you could use that model to test your image without training again

**Comment by Kitty-sunray** [comment]
Oh, wait a second. I have just connected 2+2: You are the author of "TraVeLGAN_with_perceptual_loss" code? Sorry for not recognizing your username here in first place. I sure it is very promising project that should be way better than cycleGAN, I was trying to run it for so long, and after that one [fail I wasn't able to fix](https://github.com/richardoey/TraVeLGAN_with_perceptual_loss/issues/1) I switched to this simpler version without perceptual loss. If you would kindly help to fix perceptual loss version that would be awesome! 
Also, what is your personal thoughts on TravelGAN vs [UGATIT](https://github.com/shoutOutYangJie/Morph-UGATIT)?
